src/tracer.js

```diff
@@ -228,6 +228,7 @@ export default class Tracer {
             ctx.parentId = null;
             ctx.flags = flags;
         } else {
+            this.processDeferredSampling(parent, operationName, internalTags);
```

Comment:
>Since we narrowed the scope, would't the parent in this case always be something we just serialized from the wire? The only child is going to be the server-side span.
You are right, I'll have an updated diff out shortly